extension.ts

```diff
@@ -35,13 +33,23 @@ export function deactivate(): Thenable<void> | undefined {
 	return client.stop();
 }
 
+
+
 async function spawnServer(): Promise<child_process.ChildProcess> {
-	let serverProcess = child_process.spawn('up', ['xpls', 'serve', '--verbose'])
-	// let serverProcess = child_process.spawn('')
-    serverProcess.on('error', (err: { code?: string; message: string }) => {
-		window.showWarningMessage(
+  let settings = getSettings()
+  let serverProcess = child_process.spawn(settings.upPath, ['xpls', 'serve', '--verbose'])
+  serverProcess.on('error', (err: { code?: string; message: string }) => {
+		console.error(
```

Comment:
Do you think the window warning might be more obvious to users than the console error?

Reply:
yes, absolutely, just reverted to that!